.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -99,11 +103,19 @@ runs:
       role-duration-seconds: ${{ inputs.aws-role-duration-seconds }}
       aws-region: ${{ inputs.aws-region }}
 
+  - if: inputs.publish == 'true' && inputs.aws-use-access-key == 'false'
```

Comment:
We can avoid this duplication by having `aws-access-key-id` and `aws-secret-access-key` default to an empty string, which will then use OIDC by default.

See:
 - https://github.com/aws-actions/configure-aws-credentials/blob/8a84b07f2009032ade05a88a28750d733cc30db1/index.js#L312
 - https://github.com/aws-actions/configure-aws-credentials/blob/8a84b07f2009032ade05a88a28750d733cc30db1/index.js#L291

Reply:
Good find, removed branching logic and added comments for what the default functionality is